### PR TITLE
docs: Jenkins World 2019 key takeaways blog post

### PR DESCRIPTION
**Pull request by jmcgeheeiv**
@MarckK, thank you for inviting me to contribute my LinkedIn article as a blog post.

One thing I have yet to figure out here is how to include images.

### CODE REVIEW

**Comment by MarckK** [comment]
Hi @jmcgeheeiv, Thank you for submitting this PR!  

As discussed on the phone and in email, we're very happy to publish your post; however, blog posts tend to work better when they are shorter and more focused, thus surfacing the information contained more readily for readers. At a high level: focusing on Jenkins X and reducing the length by about a third would be great. 

If you have any questions, please don't hesitate to ask here or by email!

**Comment by jmcgeheeiv** [comment]
I'm ready to move forward. Per @MarckK's request, I shortened the article by removing the sections on CDF and Kind.

File: content/news/jenkinsworld-2019-takeaways.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+share with you the dozen key things I learned about this exciting new cloud native CI/CD tool.
+Even if you never use it for CI/CD, Jenkins X provides an excellent example of how to architect 
+a cloud native application on Kubernetes.
+
+Jenkins X is a completely new CI/CD system that shares little but its name with the existing Jenkins. Jenkins X incorporates the best practices from the *State of DevOps* reports and the seminal book, *Accelerate* by [Nicole Forsgren](https://twitter.com/nicolefv), [Jez Humble](https://twitter.com/jezhumble) and [Gene Kim](https://twitter.com/RealGeneKim).
+
+<img width="70%" height="70%" alt="Capabilities of Jenkins X" src="https://www.cloudbees.com/sites/default/files/jenkinsx_capabilities.png">
+
+I attended multiple presentations by [James Strachan](https://twitter.com/jstrachan), Andrew Bayer and [James Rawlings](https://twitter.com/jdrawlings) of CloudBees, and [Mauricio Salatino](https://twitter.com/salaboy). Mr. Rawlings even gave two presentations. Each presentation covered Jenkins X from a different perspective, so here I compose everything into a unified summary.
```
Comment (MarckK) [inline content/news/jenkinsworld-2019-takeaways.md]: Add Andrew's twitter: [Andrew Bayer](https://twitter.com/abayer)

**Comment by MarckK** [comment]
The post looks fantastic, @jmcgeheeiv! Thank you so much for contributing a great write-up of Jenkins X Key Takeaways from Jenkins World 2019!

**Pull request merged**